<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorPage</class>
 <widget class="QDialog" name="DoctorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 196, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>821</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="vpl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;
</string>
     </property>
     <property name="text">
      <string>View Patients List</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>260</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;
</string>
     </property>
     <property name="text">
      <string>Write Prescriptions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;
</string>
     </property>
     <property name="text">
      <string>View Appoinment  Schedule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logoutBtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>390</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;
</string>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showbb">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;
</string>
     </property>
     <property name="text">
      <string>Search Patients Details</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>400</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;
</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>561</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 36pt &quot;Algerian&quot;;
color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Appoinment  Schedule</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>571</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <column>
      <property name="text">
       <string>Patient</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Doctor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>721</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 36pt &quot;Algerian&quot;;
color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>search patient information</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>390</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;
</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Imprint MT Shadow&quot;;
color: rgb(0, 0, 91);;</string>
     </property>
     <property name="text">
      <string>Patients Id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Mongolian Baiti&quot;;
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchBtn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>200</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>501</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 36pt &quot;Algerian&quot;;
color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Write Prescription</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 14pt &quot;Lucida Calligraphy&quot;;
background-color: rgb(43, 52, 217);
color: rgb(193, 231, 255);</string>
     </property>
     <property name="text">
      <string>Patient ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Mongolian Baiti&quot;;
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 14pt &quot;Lucida Calligraphy&quot;;
background-color: rgb(43, 52, 217);
color: rgb(193, 231, 255);</string>
     </property>
     <property name="text">
      <string>Notes/Medicines:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitBtn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>410</y>
       <width>141</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;</string>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>471</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 14pt &quot;Lucida Fax&quot;;
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>410</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 91);
font: 16pt &quot;Baskerville Old Face&quot;;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
